src/components/JobHistoryModal.vue

```diff
@@ -49,14 +60,68 @@ export default defineComponent({
     IonTitle,
     IonToolbar,
   },
+  data() {
+    return{
+      jobHistory: []
+    }
+  },
+  props: ['currentJob'],
+  computed: {
+    ...mapGetters({
+      getEnumName: 'job/getEnumName',
+      getCurrentEComStore:'user/getCurrentEComStore',
+    })
+  },
   methods: {
     closeModal() {
       modalController.dismiss({ dismissed: true });
     },
+    getTime (time: any) {
+      return DateTime.fromMillis(time).toLocaleString(DateTime.TIME_SIMPLE);
+    },
+    async fetchJobHistory() {
+      let resp;
+
+      try {
+        resp = await JobService.fetchJobInformation({
+          "inputFields": {
+            "productStoreId": this.getCurrentEComStore.productStoreId,
+            "statusId": ["SERVICE_CANCELLED", "SERVICE_CRASHED", "SERVICE_FAILED", "SERVICE_FINISHED"],
+            "statusId_op": "in",
+            "systemJobEnumId": this.currentJob?.systemJobEnumId
+          },
+          "fieldList": [ "runTime", "statusId" ],
+          "entityName": "JobSandbox",
+          "noConditionFind": "Y",
+          "viewSize": process.env.VUE_APP_VIEW_SIZE,
+          "orderBy": "runTime DESC"
+        })
+        if(resp.status === 200 && resp.data.docs?.length > 0 && !hasError(resp)) {
+          const jobs = resp.data.docs;
+          jobs.map((job: any) => {
+            job['statusDesc'] = this.store.state.util.statusDesc[job.statusId];
```

Comment:
Use getter

Reply:
Sir, updated code to add a getter instead of directly accessing state.